frontend/apps/marketing/src/components/afeEligibility/AFEEligibility.tsx

```diff
@@ -0,0 +1,187 @@
+'use client';
+
+import {useStatsigClient} from '@statsig/react-bindings';
+import {useState, useEffect} from 'react';
+
+import {LinkButton} from '@code-dot-org/component-library/button';
+import {
+  Heading2,
+  Heading3,
+  BodyTwoText,
+  BodyThreeText,
+  OverlineThreeText,
+} from '@code-dot-org/component-library/typography';
+
+import AFEForm from '@/components/afeEligibility/AFEForm';
+import AFESchoolCheck from '@/components/afeEligibility/AFESchoolCheck';
+import Link from '@/components/link';
+import {getStudioUrl} from '@/config/studio';
+
+import styles from './afeEligibility.module.scss';
+
+const SESSION_STORAGE_KEY = 'afeEligibilityData';
+
+type AFEEligibilityData = {
+  email?: string;
+  userType?: string;
+  schoolId?: string;
+  schoolName?: string;
+  isEligible?: boolean;
+  isSignedIn?: boolean;
```

Comment:
Based on screenshots, all of these are required props. Can you please make them required?

Checked locally, that works. Just make sure to update `updateEligibilityData`  to `newEligibilityData: Partial<AFEEligibilityData>`